storage/google/cloud/storage/client.py

```diff
@@ -265,8 +266,8 @@ def create_bucket(self, bucket_name, requester_pays=None, project=None):
         To set additional properties when creating a bucket, such as the
         bucket location, use :meth:`~.Bucket.create`.
 
-        :type bucket_name: str
-        :param bucket_name: The bucket name to create.
+        :type bucket_or_name: str or resource
```

Comment:
Could you make the docstring for `create_bucket` follow the same pattern as we use in BigQuery. These `:type:` and `:param:` are Sphinx/RST-style, but we are migrating to Google-style.

Docs on Google-style are at https://sphinxcontrib-napoleon.readthedocs.io/en/latest/ with an example file at https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

We had to do this for BigQuery too. Basically, as we update each method we migrate it to the Google-style.

Reply:
Updated docstring to reflect Google-style.